libbeat/processors/add_cloud_metadata/provider_google_gce.go

```diff
@@ -82,6 +94,10 @@ var gceMetadataFetcher = provider{
 				}.ApplyTo(cloud, instance)
 				trimLeadingPath("machine.type")
 				trimLeadingPath("availability_zone")
+
+				if zone := getValue("availability_zone"); len(zone) >= 2 {
```

Comment:
May you clarify what `2` means in this context? My understanding is that `zone` is a string, there is no `region` information available and we are extracting the region from the `zone` by removing the last 2 characters. 

This works as zones in GCP are `<region>-<zone>` ([docs](https://cloud.google.com/compute/docs/regions-zones#identifying_a_region_or_zone)) but to my knowledge `<zone>` is not guaranteed to be a single character.

Reply:
@endorama you are right, I assumed it from the [list of available regions/zones](https://cloud.google.com/compute/docs/regions-zones#identifying_a_region_or_zone), but I couldn't find naming convention for that.
I've updated implementation - https://github.com/elastic/beats/pull/35300/commits/cd04fee1c628da93f668c1f49fc0ae488524180a

Reply:
awesome :smiley: